web/package.json

```diff
@@ -13,6 +13,9 @@
   },
   "devDependencies": {
     "autoprefixer": "6.5.4",
+    "babel-core": "^6.25.0",
```

Comment:
I propose changing to
```
"babel-core": "^6.0.0",
"babel-loader": "^6.4.1",
```
and no babel-present-env. I believe that the 6.4.1 babel-loader series is the latest that will work with webpack 1.x which we are using.